<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreenMenu</class>
 <widget class="QWidget" name="MainScreenMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#widget_menu{
background-color: qradialgradient(spread:pad,cx: 0.5, cy: 0.5, radius: 2.3, fx: 0.5, fy: 0.5, stop:0.1 rgba(230, 230, 230, 150), stop:0.3 rgba(200,200,200, 150), stop:0.35 rgba(230,230,230, 150));
border-radius:10px;
}
QStackedWidget{
background-color: transparent;
}
QLabel{
font-family:&quot;Consolas&quot;;
font-size:30px;
}
QPushButton{
background-color: #ffffff;
border-style: outset;
border-width: 0px;
border-radius: 7px;
padding: 4px;
font-size:25px;
font-family:&quot;Consolas&quot;;
}
QPushButton:hover{
background-color: #dddddd;
}
QPushButton:pressed{
background-color: qradialgradient(spread:pad,cx: 0.5, cy: 0.5, radius: 2, fx: 0.5, fy: 0.5, stop:0.1 rgba(193, 222, 245, 255), stop:0.4 rgba(107, 148, 181, 255), stop:0.45 rgba(193, 222, 245, 255));
}
QPushButton:checked{
background-color: qradialgradient(spread:pad,cx: 0.5, cy: 0.5, radius: 2, fx: 0.5, fy: 0.5, stop:0.1 rgba(193, 222, 245, 255), stop:0.4 rgba(107, 148, 181, 255), stop:0.45 rgba(193, 222, 245, 255));
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget_menu" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>650</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>650</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget_menu">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>600</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="page_login">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="label_login">
            <property name="styleSheet">
             <string notr="true">background-color:rgba(0,0,0,0);</string>
            </property>
            <property name="text">
             <string>Login</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_login">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Select User</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>387</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_login_back">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_main_menu">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="label_main">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Main</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_recipe_menu">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Recipes</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_tools">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Tools</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_settings">
            <property name="text">
             <string>Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_logout">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Logout</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_recipes">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="label_recipe">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Recipes</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_add_recipe">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Add Recipe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_recipe_settings">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Recipe Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_recipe_back">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_tools">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="label_tools">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Tools</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_warmup">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Warmup Source</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_manual_controls">
            <property name="text">
             <string>Manual Controls</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_live_view">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Live View</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_calibrate">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Calibrate Camera</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_tools_back">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_settings">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="label_settings">
            <property name="styleSheet">
             <string notr="true">background-color:rgba(0,0,0,0);</string>
            </property>
            <property name="text">
             <string>Settings</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_hardware_settings">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string> Hardware Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_preferences">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>-1</pointsize>
             </font>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Preferences</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_user_settings">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>User Settings</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_permissions">
            <property name="text">
             <string>User Permissions</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_about">
            <property name="text">
             <string>About</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_settings_back">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_exit">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Consolas</family>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Exit To Desktop</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
